ee/clickhouse/queries/funnels/funnel_trends.py

```diff
@@ -27,8 +27,9 @@ class ClickhouseFunnelTrends(ClickhouseFunnelNew):
     In the default case, {from_step} is the first step of the funnel, meaning entrance from the very beginning.
```

Comment:
> A funnel is considered entered by a user when they have performed step {from_step} and all the steps leading to it, _in order_ (if there are any).

Is there a case where `from_step` is not the beginning of the funnel? O.o

Reply:
(sorry, hijacking PR for more clarifications on this doc)

Reply:
I've actually rewritten the description of how `from_step` and `to_step` are used so that this is clearer.

So this conflated funnel entrance with `from_stop` too much. Answering your Q:
`from_step` used for `conversion_rate` doesn't have to be beginning of the funnel – we can very well calculate what percentage of users who have reached a funnel 1→2→3→4's step 3 made it to step 4 too. A thing to keep in mind though is that regardless of `from_step`, entrance is counted using step 1, and if `from_step` is not 1, a user is only counted into `conversion_rate`'s denominator if they have reached `from_step` AND all the steps leading up to it.

Also see #4914, which exposes this feature in the API.